src/components/BusinessViewMap.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import { ExampleLocationType } from '@components/BusinessView'
+import Map from 'react-native-maps'
+
+const BusinessViewMap = ({ businessLocation } : { businessLocation : ExampleLocationType }) => (
```

Comment:
I believe you might be able to use the MapView in the business profile and set the prop showRadius to false to hide the distance radius and just have the map and pins. We can always change what gets hidden in when this prop is false if you don't want the pins to be displayed too.